proto/grp.proto

```diff
@@ -17,4 +20,45 @@ message Group {
 
   // True if this group has linked a Github token.
   bool github_linked = 4;
+
+  // Whether users from the owned domain should automatically
+  // be added to the group when they sign up (defaults to false).
+  //
+  // Not yet implemented.
+  bool auto_populate_from_owned_domain = 5;
+}
+
+message CreateGroupRequest {
+  // Group initialized with required fields except for ID.
+  //
+  // The required fields are documented in the Group proto.
+  Group group = 1;
```

Comment:
Can you double check that this works / does not create warnings since "group" is a reserved word in proto? I ran into that a little bit before (and it's why we have the unfortunately named "grp.proto").

Reply:
Seems to work - `//server` builds successfully

Reply:
👍